src/ogd/games/BLOOM/BloomLoader.py

```diff
@@ -16,6 +16,7 @@
 from ogd.core.models.enums.ExtractionMode import ExtractionMode
 from ogd.core.schemas.games.GameSchema import GameSchema
 from ogd.core.utils.utils import loadJSONFile
+from ogd.games.BLOOM.features import PersistThroughFailure
```

Comment:
This shouldn't be necessary, as we have `from ogd.games.BLOOM.features import *` above. Was there an error with PersistThroughFailure when you didn't have this line?